README.md

```diff
@@ -47,18 +47,24 @@ cmake --build . --config Release
 ```
 *Note*: Compiler with c++14 support needed. That means Visual Studio 2015 or higher on Windows. 
 
-:warning: **Note**: Viewer should be launched from same folder, where `resources` is located. 
-So you need to manualy copy `resources` to build folder, or copy executable to project root directory.
+:warning: **Note**: Viewer should be launched from the same folder, where `resources` is located. 
+So you need to manually copy `resources` to build folder, or copy executable to project root directory.
 
 ## Strategy integration
-You need special client to be able send messages in viewer. See [example C++ client](https://github.com/kswaldemar/rewind-viewer/blob/master/clients/c%2B%2B/RewindClient.h) for information about json based message
+You need a special client to be able to send messages to the viewer. See [example C++ client](https://github.com/kswaldemar/rewind-viewer/blob/master/clients/c%2B%2B/RewindClient.h) for information about JSON based message
 protocol and implement one for your language of choice.
-Also see [client examples for official local runner](https://github.com/JustAMan/russian-ai-cup-visual/tree/master/clients).
+Also, see [client examples for official local runner](https://github.com/JustAMan/russian-ai-cup-visual/tree/master/clients).
 
-Then run viewer before strategy and it should start draw frames
+Sample usage: 
+
+1. Start the viewer.
+2. Start localrunner, preferably in render_to_screen=false mode.
+3. Start your strategy of choice.
+4. To be able to drew things in the viewer you will need to create a client, send data to the client in your strategy, and **close the frame**. 
+5. There is no need to close the viewer after the strategy is done, simply start from step 2.
```

Comment:
There is no need to close the viewer after the strategy is done, simply clean old data (Edit->Clear frame data, or just Ctrl+R) and start from step 2.